ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/SchemaDescriptionFormatTest.java

```diff
@@ -0,0 +1,149 @@
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+
+public class SchemaDescriptionFormatTest {
+  ObjectMapper objectMapper = new ObjectMapper();
+
+  private void shouldSerializeCorrectly(String descriptionString,
+                                        List<FieldInfo> deserialized) throws IOException {
+    objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
this changes the state of objectMapper for all subsequent tests. 

either set this in `setUp` or, is this is specific to this test, then consider creating a new `ObjectMapper` for each test in a `setUp` method.

Reply:
An instance of the class is instantiated for each test, so this should get recreated for each test. That said, having side effects like this is bad. I'll just use local mappers